src/pkg/packaging/dir.proj

```diff
@@ -214,4 +215,82 @@
           Condition="'@(PackageProjects)' != ''" />
   </Target>
 
-</Project>
+  <Target Name="GenerateVSNugetPackages"
+          Condition="'$(OS)' == 'Windows_NT' AND '$(TargetArchitecture)' != 'arm' AND '$(TargetArchitecture)' != 'arm64'"
+          DependsOnTargets="SetupVSNugetPackages;GenerateInstallers;GenerateProjectInstallers;GenerateCombinedInstallers"
+          Inputs="@(SdkMsiComponent->'%(MsiInstallerFile)');
+                    $(NuSpecFile);
+                    $(GenerateNupkgPowershellScript)"
+          Outputs="@(SdkMsiComponent->'$(PackageOutputPath)%(ComponentId).%(ComponentVersion).nupkg')">
+
+    <Exec Command="powershell -NoProfile -NoLogo $(GenerateNupkgPowershellScript) ^
```

Comment:
Can GenerateVSNugetPackages run in parallel for every package with VS? I'd imagine that downloading nuget.exe into the same location with multiple scripts will cause corruption. I ran into this when I was implementing this in AspNetCore.

Reply:
This repo doesn't fully build in VS to my knowledge, but I'm also not sure how VS affects when things can happen in parallel.

Reply:
Let me re-clarify. The target `GenerateVSNugetPackages` is invoked each time we want to create a nuget package with a name VS.Redist.Common.*. My concern is that if the target is invoked multiple times in parallel, the download for nuget.exe will get corrupted as multiple scripts are running at the same time trying to download a file. Can these targets be invoked in parallel?

Reply:
Ah, no, they can't.